src/Components/Clock/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+type Props = {
+  clockName: number,
+};
+
+type State = {
+  time: string
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    time: new Date().toLocaleTimeString(),
+  };
+
+  timerId: NodeJS.Timer = setInterval(() => { }, 0);
+
+  componentDidMount() {
+    this.timerId = setInterval((() => this.tick()), 1000);
+  }
+
+  componentDidUpdate(old: Props) {
+    if (old.clockName !== this.props.clockName) {
+    // eslint-disable-next-line
+      console.log(`The Clock was renamed from ${old.clockName} to ${this.props.clockName}`);
+    }
+  }
```

Comment:
Check if the message appears in the console if you hide clock time
(When I hide clock time I can change the clock name, but I don't see this message on the console)